securitas/controller/group.py

```diff
@@ -62,6 +63,11 @@ def group_add_member(ipa, groupname):
             return redirect(url_for('group', groupname=groupname))
 
         flash('You got it! %s has been added to %s.' % (username, groupname), 'success')
+
+        config.conf.setup_logging()
+        msg = schema.MemberSponsorV1()
+        api.publish(api.Message(topic="hello", body={"Hello": "world!"}))
```

Comment:
if you comment out using the schema above, this message goes through to the queue when adding a new user to a group.

to view this message when  it comes in, use `fedora_messaging consume` and have ti running when messages are sent and they will print on the terminal for you